workflow-templates/code-coverage.properties.json

```
@@ -0,0 +1,9 @@
+{
+    "name": "Sonar scan Workflow",
+    "description": "Sonar scan CI starter workflow.",
+    "iconName": "SonarQube",
+    "categories": [
+        "java",
+	"nodejs"
```

```suggestion
        "java"
```
This is still a Java-only workflow, as it requires Gradle.